### PR TITLE
configs: Update default container config

### LINKED ISSUE #13497: Default configs in Docker images dont work very well out of the box
## description

The default Envoy docker image comes with an example configuration that:

- listens on `127.0.0.1:9901` with an admin interface
- proxies `http` `0.0.0.0:10000` to `https` www.google.com

Running this with

```
docker run -it --rm -p 10000:10000 -p 9901:9901 envoyproxy/envoy:v1.16.0
```

...and browsing to http://localhost:PORT neither works very well.

The admin doesnt work at all out-of-the-box because its listening to `127.0.0.1`

The google proxy creates an overlay with a spinning wheel and borks (im guessing that playing with eg `/etc/resolv.conf` it might work differently)

I would propose that we change and/or document the admin UI

For the proxy, how about we proxy to the www.envoyproxy.io website

### refs

- #13461 
- #13490

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: configs: Update default container config
Additional Description:

This updates the defaut container config to:

- use the envoyproxy.io website as the default upstream proxy
- listen for admin interface on 0.0.0.0 address

Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue] Fix #13497 
[Optional Deprecated:]

### CODE REVIEW

File: configs/envoyproxy_io_proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+admin:
```
Comment (htuch) [inline configs/envoyproxy_io_proxy.yaml:1]: Are you planning on nuking the google_com YAML? SGTM if so.

**Review by htuch — COMMENTED** [review]
LGTM

File: configs/envoyproxy_io_proxy.yaml (hunk 1 above)
Comment (phlax) [reply configs/envoyproxy_io_proxy.yaml:1]: i was wondering - which was part of the reason for marking it WIP - to reckon i should check through a bit further - and happy to remove

Comment (htuch) [reply configs/envoyproxy_io_proxy.yaml:1]: Yep, there is no good reason to be pointing at google.com, it was just arbitrary.

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**